bin/.travis/rest_functional_tests_security.yml

```diff
@@ -0,0 +1,6 @@
+
+        ezpublish_rest:
```

Comment:
this file seems to not be needed since in prepare_rest_functional_tests.sh you use sed to uncomment the Basic Auth config